Tests/ApolloInternalTestHelpers/MockSchemaMetadata.swift

```diff
@@ -116,4 +155,12 @@ public enum MockSchema2Configuration: SchemaConfiguration {
   public static func cacheKeyInfo(for type: Object, object: ObjectData) -> CacheKeyInfo? {
     CacheKeyInfo(id: "two")
   }
+  
+  public static func cacheKey(for field: Selection.Field, variables: GraphQLOperation.Variables?, path: ResponsePath) -> CacheKeyInfo? {
+    return nil
+  }
+  
+  public static func cacheKeys(for field: Selection.Field, variables: GraphQLOperation.Variables?, path: ResponsePath) -> [CacheKeyInfo]? {
```

Comment:
This hasn't been updated to the new function signature. Same above on Line 141 & in `TestMockTests`.